src/driver/oracle/remove.js

```diff
@@ -54,7 +54,7 @@ const doDelete = Promise.coroutine(function* (options) {
     const copiedIds = yield Model.find({ "data": { $ne: null } }, {_id: 1}).lean().exec();
     for(let i=0; i<copiedIds.length; i++) {
         let id = copiedIds[i]._id;
-        if(!sourceIds.get(id)) {
+        if(!sourceIds.get(id.toString())) {
```

Comment:
No need to transform id, id will always be a string (because the mongoose schema for sources defines _id as a String)